components/scream/cmake/ctest_script_scripts_tests.cmake

```diff
@@ -0,0 +1,42 @@
+cmake_minimum_required(VERSION 3.9)
+
+set(CTEST_BUILD_NAME "scripts_tests")
+
+set(CTEST_SOURCE_DIRECTORY "${SCREAM_ROOT}/scripts/tests")
+set(CTEST_BINARY_DIRECTORY "${BUILD_WORK_DIR}")
+
+if(NOT DEFINED dashboard_model)
+  set(dashboard_model Experimental)
+endif()
+if(NOT DEFINED dashboard_track)
+  set(dashboard_track E3SM_SCREAM)
+endif()
+
+set(CTEST_CMAKE_GENERATOR "Unix Makefiles")
+
+ctest_start(${dashboard_model} TRACK ${dashboard_track})
+
+ctest_configure(RETURN_VALUE CONFIG_ERROR_CODE)
+
+if (CONFIG_ERROR_CODE)
+  set (TEST_FAILS TRUE)
+else ()
+
+  # Need this code so that build errors don't get buried
+  if (DEFINED ENV{CTEST_PARALLEL_LEVEL})
```

Comment:
Since we don't use parallel level (we hard code 1), can we remove the CTEST_PARALLEL_LEVEL logic altogether?

Reply:
Good call, this was overriding the `-j1`